src/test/java/testresults/PutTestResultsCertificateGeneration.java

```diff
@@ -10,7 +10,8 @@
 import net.thucydides.core.annotations.Title;
 import net.thucydides.core.annotations.WithTag;
 import net.thucydides.junit.annotations.TestData;
-import org.apache.commons.lang3.time.DateUtils;
+import org.apache.commons.lang3.time.DateUtils;;
```

Comment:
can the extra semi colon be removed please

Reply:
Done